stream_alert_cli/athena/handler.py

```diff
@@ -199,17 +191,15 @@ def _construct_create_table_statement(schema, table_name, bucket):
         bucket=bucket)
 
 
-def create_table(athena_client, options, config):
+def create_table(options, table_type, config):
```

Comment:
I might be missing something, but is there an added benefit of passing in `table_type` as an arg if it's in `options.type`?

Reply:
the idea was this would allow for a more generic function with a better interface by accepting a string for the table_type.

the only property of `options` that was accessed within this function was `type`, which means if any caller wanted to use this function it essentially has to create a namedtuple first that would have a `type` property corresponding to the table type. it's fine to keep as is, just makes using it more difficult